log.p8

```diff
@@ -0,0 +1,92 @@
+pico-8 cartridge // http://www.pico-8.com
+version 18
+__lua__
+-- pico8lib logging library
+-- by mindfilleter
+
+-- Depends on pico8lib/strings.p8
+
+-- P8LIBLOGTZ configures the output to either use UTC or local time.
+-- Valid values are "local" or "utc"
+local P8LIBLOGTZ = "local"
+
+-- P8LIBLOGLVL configures the logging functions to only output certain log
+-- levels.
+-- 0 - disable logging
+-- 1 - errors only
+-- 2 - warnings and errors
+-- 3 - all logs (info, warnings, and errors)
+local P8LIBLOGLVL = 3
+
+--- Log the passed in object to STDOUT
+-- The passed in object will be converted to a string and then logged with a
+-- timestamp.
+-- @param level A level prefix to prepend to the message
+-- @param any The object to log
+-- @return nil
+local function log (level, any)
```

Comment:
Call this `prefix` instead of `level` since this function doesn't honor P8LIBLOGLVL.
Or move P8LIBLOGLVL enforcement to this function and... do something else for the level/string mapping?